### PR TITLE
Feature/tracker log

### PR DESCRIPTION
**Pull request by pengin7384**
Done 
- Fix undefined logs
- Change byte formula

resolve: #25

### CODE REVIEW

File: tracker-server/index.js [hunk 1]
```diff
@@ -45,11 +45,14 @@ function printNodesInfo() {
   });
   for (let i = 0; i < nodeList.length; i++) {
     const node = nodeList[i];
-    const diskAvailableMb = Math.floor(node.diskUsage.available / 1000000);
+    const diskAvailableMb = Math.floor(node.diskUsage.available / 1024 / 1024);
```
Comment (platfowner) [inline tracker-server/index.js:48]: It seem that MB is acquired by dividing by 1,000,00 and MiB is by dividing by 1024*1.24.
https://en.wikipedia.org/wiki/Megabyte

Comment (pengin7384) [reply tracker-server/index.js:48]: That's amazing... I thought `1MB = 1024KB`. I edit it!

File: tracker-server/index.js [hunk 2]
```diff
@@ -45,11 +45,14 @@ function printNodesInfo() {
   });
   for (let i = 0; i < nodeList.length; i++) {
     const node = nodeList[i];
-    const diskAvailableMb = Math.floor(node.diskUsage.available / 1000000);
-    const memoryFreeMb = Math.round(node.memoryUsage.free / 1024 / 1024);
+    const diskAvailableMb = Math.floor(node.diskUsage.available / 1000 / 1000);
+    const memoryFreeMb = Math.round(node.memoryUsage.free / 1000 / 1000);
```
Comment (minsulee2) [inline tracker-server/index.js:49]: Question: What makes it difference from / 1024 / 1024?

File: tracker-server/index.js (hunk 1 above)
Comment (platfowner) [reply tracker-server/index.js]: Thanks!

File: tracker-server/index.js (hunk 2 above)
Comment (platfowner) [reply tracker-server/index.js:49]: The problem is the difference gets bigger and bigger as the unit become bigger.
So it looks like people use MB and MiB more carefully nowadays: https://en.wikipedia.org/wiki/Megabyte

Comment (pengin7384) [reply tracker-server/index.js:49]: Some use 1000 and some use 1024... and it created confusion. It seems to created a MiB to solve this problem

**Comment by pengin7384** [comment]
Thanks for reviews!

**Pull request merged**